cli/climerger.php

```diff
@@ -34,8 +34,40 @@
 require_once $CFG->dirroot . '/lib/clilib.php';
 require_once __DIR__ . '/../lib/autoload.php';
 
+// Now get cli options.
+list($options, $unrecognized) = cli_get_params(
+    array(
+        'debugdb'    => false,
+        'alwaysRollback' => false,
+        'help'    => false,
+    )
+);
+
+if ($unrecognized) {
+    $unrecognized = implode("\n  ", $unrecognized);
+    cli_error(get_string('cliunknowoption', 'admin', $unrecognized), 2);
+}
+
+if ($options['help']) {
+    $help =
+        "Create alternative component cache file
```

Comment:
Should it be like this? `Create alternative component cache file` is not an expected message for a climerger ;-)

Suggestion: `Command LIne user merger. These are the available options:`